include/Config.h

```diff
@@ -167,24 +183,25 @@ class Config {
   void dump() const {
     Logger::debug() << "Config>\n"
     << "\t" << "bitcode_file_list: " << bitcodeFileList << '\n'
+    << "\t" << "dynamic_library_file_list: " << dynamicLibraryFileList << '\n'
     << "\t" << "project_name: " << getProjectName() << '\n'
     << "\t" << "test_framework: " << getTestFramework() << '\n'
     << "\t" << "distance: " << getMaxDistance() << '\n'
     << "\t" << "dry_run: " << isDryRun() << '\n'
     << "\t" << "fork: " << getFork() << '\n';
 
-    if (getMutationOperators().size() > 0) {
+    if (mutationOperators.empty() == false) {
       Logger::debug() << "\t" << "mutation_operators: " << '\n';
 
-      for (auto mutationOperator : getMutationOperators()) {
+      for (auto mutationOperator : mutationOperators) {
         Logger::debug() << "\t- " << mutationOperator << '\n';
       }
     }
-
-    if (getTests().empty() == false) {
```

Comment:
For a future: please avoid doing changes that are not directly relevant to a subject of your PRs. Those changes become a noise for reviewers of your code. Let's always try to do separate PRs (smaller - better).